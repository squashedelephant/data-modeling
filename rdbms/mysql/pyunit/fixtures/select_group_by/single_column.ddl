CREATE TABLE forecast(id INT NOT NULL AUTO_INCREMENT, year INT, quarter CHAR(2), sales DECIMAL(8,2), PRIMARY KEY (id));
