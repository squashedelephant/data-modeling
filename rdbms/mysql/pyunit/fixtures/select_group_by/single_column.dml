INSERT INTO forecast(year, quarter, sales) VALUES(2017, 'q1', 50000.00);
INSERT INTO forecast(year, quarter, sales) VALUES(2017, 'q2', 40000.00);
INSERT INTO forecast(year, quarter, sales) VALUES(2017, 'q3', 50000.00);
INSERT INTO forecast(year, quarter, sales) VALUES(2017, 'q4', 75000.00);
