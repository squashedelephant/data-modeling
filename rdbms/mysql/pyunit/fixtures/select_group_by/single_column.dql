SELECT year, SUM(sales) AS annual_sales FROM forecast GROUP BY year ASC;
