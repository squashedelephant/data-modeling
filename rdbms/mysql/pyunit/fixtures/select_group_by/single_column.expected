2017
215000.00
